non-atomic-action-interpreter-module/README.md

```diff
@@ -0,0 +1,135 @@
+# Non-Atomic Action Interpreter Module
```

Comment:
looks like this component cannot be installed with conan
why isn't this supported?

Reply:
This component is not a library, it is a module, that can be linked as dlopen-library only. Conan is aimed to manage libraries only.